modules/devguide/examples/CloudConnect.java

```diff
@@ -14,97 +14,82 @@
-import com.couchbase.client.core.env.IoConfig;
+// Required for Capella connection configuration
 import com.couchbase.client.core.env.SecurityConfig;
-import com.couchbase.client.java.Bucket;
-import com.couchbase.client.java.Cluster;
-import com.couchbase.client.java.ClusterOptions;
-import com.couchbase.client.java.Collection;
-import com.couchbase.client.java.Scope;
 import com.couchbase.client.java.env.ClusterEnvironment;
-import com.couchbase.client.java.json.JsonArray;
-import com.couchbase.client.java.json.JsonObject;
-import com.couchbase.client.java.manager.query.CreatePrimaryQueryIndexOptions;
-import com.couchbase.client.java.query.QueryResult;
-import com.couchbase.client.java.query.QueryScanConsistency;
+// end::capella-imports[]
 
-public class CloudConnect {
-
-    public static void main(String... args) {
-        // tag::cloud-connect[]
-        // Update these variables to point to your Couchbase Capella instance and credentials.
-        String endpoint = "--your-instance--.dp.cloud.couchbase.com";
-        String bucketname = "bucketname";
-        String username = "username";
-        String password = "password";
-
-        String tlsCertificate = "-----BEGIN CERTIFICATE-----\n" +
-          "... your certificate content in here ...\n" +
-          "-----END CERTIFICATE-----";
-        // User Input ends here.
-
-        // Configure TLS
-        List<X509Certificate> cert = SecurityConfig.decodeCertificates(Collections.singletonList(tlsCertificate));
-        SecurityConfig.Builder securityConfig = SecurityConfig
-          .enableTls(true) // Enable transport security
-          .trustCertificates(cert); // Configure the cloud certificate
-          // During development, if you want to trust all certificates you can connect
-          // to your cloud like with the InsecureTrustManagerFactory. As the name points
-          // out, this is INSECURE!
-          // .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)
 
-        // Build the environment with the TLS config
-        ClusterEnvironment env = ClusterEnvironment
-          .builder()
-          .securityConfig(securityConfig)
-          .build();
-
-        // Initialize the Connection
-        Cluster cluster = Cluster.connect(endpoint, ClusterOptions.clusterOptions(username, password).environment(env));
-        Bucket bucket = cluster.bucket(bucketname);
-        bucket.waitUntilReady(Duration.parse("PT10S")) ;
-        Collection collection = bucket.defaultCollection();
+public class CloudConnect {
+  // Update these variables to point to your Couchbase Capella instance and credentials.
+  static String endpoint = "cb.njg8j7mwqnvwjqah.cloud.couchbase.com";
+  static String bucketname = "travel-sample";
+  static String username = "username";
+  static String password = "Password!123";
+  
+  public static void main(String... args) {
+    // tag::connect[]
+    // Configure TLS
+    SecurityConfig.Builder securityConfig = SecurityConfig
```

Comment:
Do we still need this? Thought this was automatic now for Capella connections?

Reply:
Works for me in Go without a Security config, that's why I ask. 
Not sure if the same or different for Java 😕 .

Reply:
Don't know - I removed the Certificate wrangling, but I thought you still had to ensure it was TLS? @daschl to confirm?

Reply:
Ah ok. Lemme try without, even better if so...

Reply:
`An exception occured while executing the Java class. The DNS SRV record for 'cb.njg8j7mwqnvwjqah.cloud.couchbase.com' indicates TLS must be used when connecting to this cluster. Please enable TLS by setting the 'security.enableTLS' client setting to true. If the Cluster does not use a shared ClusterEnvironment, an alternative way to enable TLS is to prefix the connection string with "couchbases://" (note the final 's')`

Reply:
Yup, works great with `couchbases://` in connection string...
That's definitely the way to go for Hello World example, thanks!

Reply:
(also nice error message Michael & co!)